Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6
+
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN pip install -r requirements.txt
+
+EXPOSE 5000 
+
+CMD flask run --host=0.0.0.0
```

Comment:
prefer `python app.py`, think this does some work under the hood that we may not want (env vars) -- though we'll probably do gunicorn soon anyway